src/core/core.helpers.js

```diff
@@ -723,7 +723,7 @@ module.exports = function(Chart) {
 	};
 	helpers.addEvent = function(node, eventType, method) {
 		if (node.addEventListener) {
-			node.addEventListener(eventType, method);
+			node.addEventListener(eventType, method, {passive: true});
```

Comment:
Is there any browser detection function which is already in Chart.js code? Then we could use that. I have found `Chart.platform` but I don't think it can be used to detect IE and Edge. Any thoughts?